database/README.md

```diff
@@ -7,148 +7,98 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/README.
-
-1.  the chart file
-2.  the header part of all dimension files
-3.  if the collected metrics are stored far enough in the dimension file, another page will become dirty, for each
-    dimension
-
-Each page in Linux is 4KB. So, with 200 charts and 1000 dimensions, there will be 1200 to 2200 4KB pages dirty pages
-every second. Of course 1200 of them will always be dirty (the chart header and the dimensions headers) and 1000 will be
-dirty for about 1000 seconds (4 bytes per metric, 4KB per page, so 1000 seconds, or 16 minutes per page).
-
-Hopefully, the Linux kernel does not sync all these data every second. The frequency they are synced is controlled by
-`/proc/sys/vm/dirty_expire_centisecs` or the `sysctl` `vm.dirty_expire_centisecs`. The default on most systems is 3000
-(30 seconds).
-
-On a busy server centralizing metrics from 20+ servers you will experience this:
-
-![image](https://cloud.githubusercontent.com/assets/2662304/23834750/429ab0dc-0764-11e7-821a-d7908bc881ac.png)
+You can select the database mode by editing `netdata.conf` and setting:
 
-As you can see, there is quite some stress (this is `iowait`) every 30 seconds.
+```conf
+[db]
+  # dbengine (default), ram, save (the default if dbengine not available), map (swap like), none, alloc
+  mode = dbengine
+```
 
-A simple solution is to increase this time to 10 minutes (60000). This is the same system with this setting in 10
-minutes:
+## Netdata Longer Metrics Retention
 
-![image](https://cloud.githubusercontent.com/assets/2662304/23834784/d2304f72-0764-11e7-8389-fb830ffd973a.png)
+Metrics retention affects 3 parameters on the operation of a Netdata Agent:
 
-Of course, setting this to 10 minutes means that data on disk might be up to 10 minutes old if you get an abnormal
-shutdown.
+1. The disk space required to store the metrics.
+2. The memory the Netdata Agent will require to have that retention available for queries.
+3. The CPU resources that will be required to query longer time-frames.
 
-There are 2 more options to tweak:
+As retention increases, the resources required to support that retention increase too.
 
-1.  `dirty_background_ratio`, by default `10`.
-2.  `dirty_ratio`, by default `20`.
+Since Netdata Agents usually run on the edge, inside production systems, Netdata Agent **parents** should be considered.
+When having a **parent - child** setup, the child (the Netdata Agent running on a production system) delegates all its
+functions, including longer metrics retention and querying, to the parent node that can dedicate more resources to this
+task. A single Netdata Agent parent can centralize multiple children Netdata Agents (dozens, hundreds, or even thousands
+depending on its available resources).
 
-These control the amount of memory that should be dirty for disk syncing to be triggered. On dedicated Netdata servers,
-you can use: `80` and `90` respectively, so that all RAM is given to Netdata.
+## Running Netdata in embedded devices
```

Comment:
This whole section is problematic. We're not even saying here which memory mode they need to run with. 
I suggest we move this to the "high performance netdata" and suggest the following:

- If your devices can stream data to a parent without incurring significant traffic costs, use memory mode RAM and set up streaming
- If cost of data transfer is a concern, use memory mode save or map (to tell you the truth I never understood myself which is appropriate for which use case). 

You'll find that we are already suggesting lower retention in the high performance netdata doc.